varya/style.css

```diff
@@ -364,16 +400,30 @@ body[class*="woocommerce"] #page .woocommerce button.button,
 body[class*="woocommerce"] #page .woocommerce input.button,
 body[class*="woocommerce"] #page .woocommerce .widget_shopping_cart .buttons a,
 body[class*="woocommerce"] #page .woocommerce.widget_shopping_cart .buttons a {
+	line-height: 1;
 	line-height: var(--button--line-height);
+	color: #FFFFFF;
 	color: var(--button--color-text);
 	cursor: pointer;
+	font-weight: normal;
 	font-weight: var(--button--font-weight);
+	font-family: 'Fira Sans'
+,
+Helvetica
+,
+Arial
+,
+sans-serif;
```

Comment:
@jffng, something weird is happening here and I’m not sure why but it seems like an issue with `postcss-custom-properties`. This likely happens in a few other places where strings or CSS values have commas. Might be worth it to run the post-css linting _after_ the `postcss-custom-properties` process so it gets fixed automatically, but I’m not sure that’s possible.